java/java.hints/test/unit/src/org/netbeans/modules/java/hints/jdk/ConvertSwitchToRuleSwitchTest.java

```diff
@@ -875,6 +875,73 @@ public void testSwitch2SwitchExpressionNestedInnerSwitchStatement() throws Excep
                               "     }\n" +
                               "}\n");
     }
+    
+    public void testSwitchToRuleSwitchFormattingMultiple() throws Exception {
+        if(!ConvertSwitchToRuleSwitchTest.isJDK14())
+            return;
+        String output = HintTest.create()
+                .input("package test;" +
+                        "public class Test {\n" +
+                        "     private void test(int p) {\n" +
+                        "         String result;\n" +
+                        "         switch (p) {\n" +
+                        "            case 0:\n" +
+                        "            case 1:\n" +
+                        "            case 2:\n" +
+                        "            case 3: result=\"a\"; break;\n" +
+                        "            default: System.err.println(\"No.\"); break;" +
+                        "         }\n" +
+                        "     }\n" +
+                        "}\n")
+                .sourceLevel(SourceVersion.latest().name())
+                .run(ConvertSwitchToRuleSwitch.class)
+                .findWarning("3:9-3:15:verifier:" + Bundle.ERR_ConvertSwitchToRuleSwitch())
+                .applyFix()
+                .assertCompilable().getOutput();
+        String expected="package test;" +
+                        "public class Test {\n" +
+                        "     private void test(int p) {\n" +
+                        "         String result;\n" +
+                        "         switch (p) {\n" +
+                        "            case 0, 1, 2, 3 -> result=\"a\";\n" +
+                        "            default -> System.err.println(\"No.\");\n" +
+                        "         }\n" +
+                        "     }\n" +
+                        "}\n";
+        assertEquals(expected, output);
```

Comment:
A partly stylistic comment there: it is usually better to have testcases for the code generation under `java/java.source.base/test/unit/src/org/netbeans/api/java/source/gen`, so that regressions are found without running `java.hints` tests. The `java.hints` tests then can usually use `assertOutput`, which ignores formatting changes, but if formatting check is desired, `assertVerbatimOutput` can be used instead of calling `getOutput()` and `assertEquals`.

Reply:
Thanks, have added the regression test in java/java.source.base/test/unit/src/org/netbeans/api/java/source/gen and made changes to java.hints test.